src/pages/InvoiceDetail.js

```diff
@@ -0,0 +1,97 @@
+// @ts-nocheck
+import { useNavigation } from "@react-navigation/native";
+import React from "react";
+import { Text, View, TouchableOpacity } from "react-native";
+import {styles} from '../resources/Styles'
+
+const InvoiceDetail = ({navigation, route}) => {
+    const {invoice} = route.params
+    console.log(invoice)
+    const detailButton = "\u27A4"
+    const payed= "\u2705"
+    const pending = "\u231B"
+    const backButtonChar = "\u276e"
+
+    function formatDateInput(date){
+        if(typeof date == "string"){
+            let date = Date.parse(date)
+        }
+        let localDate = new Date(date)
+        return localDate.toLocaleDateString("de-de")
+    }
+
+   function payedSymbol(payedAt){
+        if(payedAt.Valid){
+            return payed
+        }
+        else {
+            return pending
+        }
+    }
+    function payedText(payedAt){
+        if(payedAt.Valid){
+            return "Bezahlt am " + formatDateInput(payedAt.Time)
+        }
+        else {
+            return "Ausstehend"
+        }
+    }
+
+    function goBack(){
```

Comment:
Ist es jetzt besser das "goBack" so wie hier zu implementieren oder über den Button in App.js?

Reply:
Hier kann man ein Customizing betreiben und in einen Header einbauen. Sonst kann man das Aussehen davon nicht ändern